app/js/native-objects-extension.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+Object.isEmpty = function(object) { return ! Object.keys(object).length; };
```

Comment:
Oh. Extending natives is one thing, extending `Object` is another, and doing it in the browser is… yet another. Let's discuss this this afternoon.